crates/optimism/primitives/src/transaction/signed.rs

```diff
@@ -44,8 +45,14 @@ pub struct OpTransactionSigned {
     #[deref]
     #[as_ref]
     pub transaction: OpTypedTransaction,
+
+    /// Can we attach a conditional the moment a transaction is deserialized?
+    pub conditional: Option<TransactionConditional>
```

Comment:
this we shouldn't do, because this is a representation fo the actual tx type and shouldn't be annotated with out of protocol data